### DISCUSSION TITLE
How do I download the multiple files uploaded as array to model

### DISCUSSION THREAD

**Discussion by tudorr89** [filamentphp/filament#11505] [Help] [answered]
### Package

Table builder

### Package Version

3.0-stable

### How can we help you?

Hello guys! Thanks for the awesome work done and exucuse me for the noob question. I have multiple file upload assigned to a model already cast to an array. 

How can I link all the files for the specific model as an array of links to download?

Apologies in advance, i could not find a solution.

Regards,
Tudor

**Comment by leandrocfe** [comment, marked as answer]
Maybe using `->state()`? https://filamentphp.com/docs/3.x/tables/columns/getting-started#calculated-state
You can access the model via `$record` and return a custom html with all links.

**Reply by tudorr89** [reply]
Thanks for the info! Indeed, it was the easiest way to achieve this
